README.md

```diff
@@ -40,7 +40,7 @@ defmodule Person do
   use Csv.Schema
   alias Csv.Schema.Parser
 
-  schema "path/to/person.csv" do
+  schema path: "path/to/person.csv" do
```

Comment:
Could we do it backwards-compatible? So that if you do not specify anything it is the same as specifing `path`

Reply:
It's already backward-compatible because of the last match in csv_schema.ex that works as the older version